consensus/src/quorum_store/batch_coordinator.rs

```diff
@@ -35,11 +41,9 @@ pub struct BatchCoordinator {
     batch_store_tx: Sender<BatchStoreCommand>,
     proof_coordinator_tx: Sender<ProofCoordinatorCommand>,
     max_batch_bytes: usize,
-    // Local
-    batch_aggregator: BatchAggregator,
-    fragment_id: usize,
-    // Remote
-    batch_aggregators: HashMap<PeerId, BatchAggregator>,
+    remote_batch_aggregators: HashMap<PeerId, BatchAggregator>,
+    local_batch_aggregator: BatchAggregator,
```

Comment:
Do we have many batch coordinators now?
If yes, why do all have local batch aggregator?

Reply:
I think it's a bit cleaner to just have local_batch_aggregator instead of using the hashmap, because the functions that handle remote and local are entirely different.